alerter/test/data_transformers/node/test_cosmos.py

```diff
@@ -175,6 +176,12 @@ def setUp(self) -> None:
                 }
             }
         }
+
+        self.raw_data_example_result_mev = copy.deepcopy(
+            self.raw_data_example_result_all)
+        self.raw_data_example_result_mev['tendermint_rpc']['result']['data']['is_peered_with_sentinel'] = True
+        self.raw_data_example_result_mev['tendermint_rpc']['result']['meta_data']['is_mev_tendermint_node'] = True
```

Comment:
Since it is being defined over here...

Reply:
(see https://github.com/SimplyVC/panic/pull/380/files#r1080991742)

Reply:
I am redefining here, as this is a new format of raw data for use in testing that the transformer understands mev-data if it is present.